### ISSUE TITLE
Nightly builds website is broken.

### ISSUE THREAD

**Issue by rrroyal** [iina/iina-website#22]
[Nightly builds website](https://nightly.iina.io) just shows the message "**It works!**" without any styles or other HTML elements. Nothing else.

**Comment by saagarjha** [comment]
I guess it does not in fact work :P I know this was fine until recently, I think @HarukaMa runs this?

**Comment by HarukaMa** [comment]
On 2020-10-10 07:48, Saagar Jha wrote:

> I guess it does not in fact work :P I know this was fine until 
> recently, I think @HarukaMa [1] runs this?
> 
> --
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub [2], or unsubscribe 
> [3].

I did upgrade the server to 10.15 to make it able to build newer 
versions, but I didn't check if the redirection from iina.io still works 
(I only checked it by directly accessing the server). Apparently some 
parts of configuration was not correctly migrated. Will check soon.

Links:
------
[1] https://github.com/HarukaMa
[2] 
https://github.com/iina/iina-website/issues/22#issuecomment-706433285
[3]
